CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+
+## [v2.0.0](https://github.com/ably/kafka-connect-ably/tree/v2.0.0) (2022-04-20)
+
+[Full Changelog](https://github.com/ably/kafka-connect-ably/compare/v2.0.0...HEAD)
+
+**Implemented enhancements:**
+
+- Optimize integration tests [\#58](https://github.com/ably/kafka-connect-ably/issues/58)
+- Remove client.use.binary.protocol configuration  [\#57](https://github.com/ably/kafka-connect-ably/issues/57)
+- Add configuration validators [\#53](https://github.com/ably/kafka-connect-ably/issues/53)
+- Make message name configurable and interpolable [\#46](https://github.com/ably/kafka-connect-ably/issues/46)
+- Code reorganization and refinement [\#63](https://github.com/ably/kafka-connect-ably/pull/63) ([ikbalkaya](https://github.com/ikbalkaya))
+- Add distributed mode configuration file [\#62](https://github.com/ably/kafka-connect-ably/pull/62) ([ikbalkaya](https://github.com/ikbalkaya))
+- Remove client.use.binary.protocol configuration [\#60](https://github.com/ably/kafka-connect-ably/pull/60) ([ikbalkaya](https://github.com/ikbalkaya))
+- Add .editorconfig for IntelliJ formatting support [\#54](https://github.com/ably/kafka-connect-ably/pull/54) ([ikbalkaya](https://github.com/ikbalkaya))
```

Comment:
Changes like this don't benefit users of the plugin so we tend not to include them in changelogs. I would suggest you exclude entries like this and focus instead on customer value.

Reply:
b919c3fda69c9ece71804f600f95424509177a6c

Reply:
Also removed a bit more entries here 1c9357f4417321fd208c53a7f525050a2fdff6d4 and tried to limit to entries which should be useful for customers